### PR TITLE
[MRG] Auto refresh with html

### PR DESCRIPTION
**Pull request by Lamaun**
I think the current solution is not very user friendly.

### CODE REVIEW

File: excalibur/www/templates/job.html [hunk 1]
```diff
@@ -62,10 +62,11 @@ <h4>Extracted Data</h4>
     {% endfor %}
   </div>
 {% else %}
+  <meta http-equiv="refresh" content="5" >
```
Comment (jnothman) [inline excalibur/www/templates/job.html]: Is a `meta` tag allowed in the body? I also wouldn't wait as long as 5 seconds.

An alternative is:
```suggestion
<script type="text/javascript">setTimeout('location.reload()', 500)</script>
```
(for .5 seconds)

Comment (Lamaun) [reply excalibur/www/templates/job.html]: In my use case 5 seconds is about right. Don't forget that every new get request is a small load to the server. Chrome allows the meta tag to be in the body. Not tested with other browsers.
I personally like to minimize javascript, but that's just personal preference.

Comment (jnothman) [reply excalibur/www/templates/job.html]: I don't think the load on the server to do a couple of db lookups, for a private server, is a huge deal. 5s is a lot for many operations.

Comment (vinayak-mehta) [reply excalibur/www/templates/job.html]: I agree with @jnothman. .5s should be a good reload interval. @Lamaun can you please change it, will merge the PR after that.

**Comment by Lamaun** [comment]
Done

**Comment by vinayak-mehta** [comment]
LGTM!

**Pull request merged**